skbio/diversity/_fast_base_cy.pyx

```diff
@@ -0,0 +1,215 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+cimport numpy as np
+cimport cython
```

Comment:
This is the normal form as I've seen it (iirc also how we do in biom);
remember, cython is a different language with special handling for numpy.
Though a separate alias isn't a problem.

BTW, if you haven't seen it, cythons annotate is very useful and relevant
for the questions coming up
On Sep 9, 2015 4:38 PM, "mortonjt" <notifications@github.com> wrote:

> In skbio/diversity/_fast_base_cy.pyx
> <https://github.com/biocore/scikit-bio/pull/1040#discussion_r39042358>:
>
> > @@ -0,0 +1,215 @@
> > +# ----------------------------------------------------------------------------
> > +# Copyright (c) 2013--, scikit-bio development team.
> > +#
> > +# Distributed under the terms of the Modified BSD License.
> > +#
> > +# The full license is in the file COPYING.txt, distributed with this software.
> > +# ----------------------------------------------------------------------------
> > +
> > +import numpy as np
> > +cimport numpy as np
> > +cimport cython
>
> Also need to be careful here, the cimport numpy as np will overwrite import
> numpy as np
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1040/files#r39042358>.
>